spec/views/users/users_show_spec.rb

```diff
@@ -1,32 +1,42 @@
 require 'rails_helper'
 
 RSpec.describe 'User', type: :feature do
-  describe 'content of user index page' do
-    let(:user1) { User.create(name: 'Juwairiyya', photo_link: 'photo_url', bio: 'text', posts_counter: 0) }
-
+  describe '#show users index' do
     before(:each) do
-      user1.posts.create(title: 'Post Title', text: 'Post Content')
-      visit "/users/#{user1.id}"
+      @user = User.create(name: 'Juwairiyya Sadiq', bio: 'Undisputed Queen of DIY', photo_link: 'photo',
+                          posts_counter: 0)
+      @post4 = Post.create(title: 'Hello', text: 'How to make a table', author_id: @user.id, comments_counter: 0,
+                           likes_counter: 0)
+      @post3 = Post.create(title: 'Hi', text: 'How to stay healthy', author_id: @user.id, comments_counter: 0,
+                           likes_counter: 0)
+      @post2 = Post.create(title: 'Health recipe', text: 'Eat balance diet', author_id: @user.id, comments_counter: 0,
+                           likes_counter: 0)
+      @post1 = Post.create(title: 'Energy', text: 'How to set the solar system', author_id: @user.id,
+                           comments_counter: 0, likes_counter: 0)
+
+      visit user_path(@user)
```

Comment:
- [ ] All the tests in your `users_show_spec.rb` are failing as a result of your `before`block. shown in the attached image below 👇 
![image](https://github.com/Zuwairanajma/Blog_App/assets/46839548/e845db56-3566-4493-aa50-0697b701b753)

Reply:
Hello @codepantha,  
Thanks for the precise review, we pushed a few commits and corrected the issue.
Reference the test result below:
![Rspec Testing](https://github.com/Zuwairanajma/Blog_App/assets/106805225/0ba8e5df-4e8a-47e0-bbb2-adb305077f8a)